Compressonator_Root.props

```diff
@@ -16,6 +16,7 @@
     <Compressonator_OPENEXR>$(SolutionDir)..\..\..\..\Common\Lib\Ext\OpenEXR\openexr-2.2.0\</Compressonator_OPENEXR>
     <Compressonator_ILMBASE>$(SolutionDir)..\..\..\..\Common\Lib\Ext\OpenEXR\ilmbase-2.2.0\</Compressonator_ILMBASE>
     <Compressonator_OPENGL>$(SolutionDir)..\..\..\..\Common\Lib\Ext\OpenGL\</Compressonator_OPENGL>
+    <Compressonator_RAPIDXML>$(SolutionDir)..\..\..\External\rapidxml\</Compressonator_RAPIDXML>
```

Comment:
Please use ..\..\..\..\Common\Lib\Ext\rapidxml 

This source uses Boost and MIT licences, which is ok if used as user downloaded code in common.  MIT licenses is highly preferred.

Reply:
I don't have the bandwidth to work on this project anymore. After the refactor to VS2017 I no longer know how to build and test changes. I had been working for a few months on proper CMake integration which added better cross platform support. In my opinion, if this project is intended to be a cross platform product then it needs to be using CMake and any Visual Studio artifacts should be deleted. I think you have the power to make these changes so if you're interested, I'm happy for you to do so, if not, you can decline this pull request.

My [feature/cpp17](https://github.com/UebelAndre/Compressonator/tree/feature/cpp17) branch has all the CMake migration work in it as well as MacOS support for the GUI as well. I'll leave it up for some time but will likely delete it in the future as it becomes more and more detached form HEAD of master. I think this project has a lot of potential but there's a lot of cleanup that needs to be done to make it maintainable.